{"uid":"3c5391cded79b2ea","name":"test_network_cut","fullName":"tests.integration.ha_tests.test_self_healing#test_network_cut","historyId":"6b9f6734f6f55bf616f6dd52eb37e3bd","time":{"start":1714616628613,"stop":1714616932805,"duration":304192},"description":"Completely cut and restore network.","descriptionHtml":"<p>Completely cut and restore network.</p>\n","status":"failed","statusMessage":"AssertionError: Unable to create continuous_writes table","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 849, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 883, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 256, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 182, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 172, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1772, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 195, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 390, in test_network_cut\n    await start_continuous_writes(ops_test, app)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/tests/integration/ha_tests/helpers.py\", line 688, in start_continuous_writes\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(3), reraise=True):\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 347, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 325, in iter\n    raise retry_exc.reraise()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 158, in reraise\n    raise self.last_attempt.result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/tests/integration/ha_tests/helpers.py\", line 696, in start_continuous_writes\n    assert action.results[\"result\"] == \"True\", \"Unable to create continuous_writes table\"\nAssertionError: Unable to create continuous_writes table","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1714613649177,"stop":1714613649177,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1714613649177,"stop":1714613649177,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1714613649177,"stop":1714613649325,"duration":148},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout","time":{"start":1714614930128,"stop":1714614932203,"duration":2075},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1714616628612,"stop":1714616628612,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1714616628612,"stop":1714616628613,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1714616932806,"stop":1714617236352,"duration":303546},"status":"failed","statusMessage":"AssertionError: Unable to clear up continuous_writes table\nassert 'False' == 'True'\n  \n  - True\n  + False\n","statusTrace":"  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 302, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 294, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/tests/integration/ha_tests/conftest.py\", line 29, in continuous_writes\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(3), reraise=True):\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 347, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 325, in iter\n    raise retry_exc.reraise()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 158, in reraise\n    raise self.last_attempt.result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/tests/integration/ha_tests/conftest.py\", line 37, in continuous_writes\n    assert action.results[\"result\"] == \"True\", \"Unable to clear up continuous_writes table\"\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1714617236353,"stop":1714617236353,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout::finalizer","time":{"start":1714617839975,"stop":1714617840003,"duration":28},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1714617840004,"stop":1714617840522,"duration":518},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1714617840523,"stop":1714617840523,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1714617840523,"stop":1714617840523,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1714617840523,"stop":1714617840523,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"fv-az1532-449"},{"name":"thread","value":"7235-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":5,"skipped":0,"passed":11,"unknown":0,"total":17},"items":[{"uid":"2fdce148145c2d5b","reportUrl":"../30//#testresult/2fdce148145c2d5b","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/3","time":{"start":1714528687579,"stop":1714528994434,"duration":306855}},{"uid":"b3b484514afdde9f","reportUrl":"../28//#testresult/b3b484514afdde9f","status":"passed","time":{"start":1714442428610,"stop":1714442763001,"duration":334391}},{"uid":"bea65760f7ba2f45","reportUrl":"../27//#testresult/bea65760f7ba2f45","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f0070762740 state=finished raised AssertionError>]","time":{"start":1714358603449,"stop":1714358795287,"duration":191838}},{"uid":"95d6449affef43c9","reportUrl":"../26//#testresult/95d6449affef43c9","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/3","time":{"start":1714269637556,"stop":1714269742212,"duration":104656}},{"uid":"a2fb5175b3f40ba8","reportUrl":"../22//#testresult/a2fb5175b3f40ba8","status":"passed","time":{"start":1714010442186,"stop":1714010908206,"duration":466020}},{"uid":"15ce1282555f1c8c","reportUrl":"../21//#testresult/15ce1282555f1c8c","status":"passed","time":{"start":1713925044370,"stop":1713925550847,"duration":506477}},{"uid":"33954c9de4e92453","reportUrl":"../20//#testresult/33954c9de4e92453","status":"passed","time":{"start":1713839143808,"stop":1713839625307,"duration":481499}},{"uid":"f0acba7813aa3b59","reportUrl":"../19//#testresult/f0acba7813aa3b59","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f73bb519960 state=finished raised AssertionError>]","time":{"start":1713755211751,"stop":1713755392632,"duration":180881}},{"uid":"4d1d82cac2b3b63d","reportUrl":"../18//#testresult/4d1d82cac2b3b63d","status":"passed","time":{"start":1713664692999,"stop":1713665151527,"duration":458528}},{"uid":"8ccbe64ed8799be0","reportUrl":"../15//#testresult/8ccbe64ed8799be0","status":"passed","time":{"start":1713492043561,"stop":1713492565161,"duration":521600}},{"uid":"10d281eedf1f4a58","reportUrl":"../14//#testresult/10d281eedf1f4a58","status":"passed","time":{"start":1713407655662,"stop":1713408130521,"duration":474859}},{"uid":"fc01af3b5cff573f","reportUrl":"../13//#testresult/fc01af3b5cff573f","status":"passed","time":{"start":1713319794050,"stop":1713320315887,"duration":521837}},{"uid":"5cb949d5d24176b2","reportUrl":"../8//#testresult/5cb949d5d24176b2","status":"passed","time":{"start":1713232921942,"stop":1713233346920,"duration":424978}},{"uid":"c908cad1a0454f3e","reportUrl":"../7//#testresult/c908cad1a0454f3e","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7fee5376e9b0 state=finished raised AssertionError>]","time":{"start":1713152186797,"stop":1713152369013,"duration":182216}},{"uid":"988fbdd3958d0436","reportUrl":"../4//#testresult/988fbdd3958d0436","status":"passed","time":{"start":1712973749030,"stop":1712974349305,"duration":600275}},{"uid":"d582975922cfb4b5","reportUrl":"../2//#testresult/d582975922cfb4b5","status":"passed","time":{"start":1712887978904,"stop":1712888498517,"duration":519613}}]},"tags":["asyncio","group(1)"]},"source":"3c5391cded79b2ea.json","parameterValues":[]}